### PR TITLE
Make browser.js ES5 compliant

### PR DESCRIPTION
**Pull request by dhalbrook**
Rewriting browser.js function to be ES5 compliant.  See https://github.com/pinojs/pino/issues/656

### CODE REVIEW

File: browser.js [hunk 1]
```diff
@@ -307,9 +307,11 @@ function noop () {}
 /* eslint-disable */
 /* istanbul ignore next */
 function pfGlobalThisOrFallback () {
-  const defd = (o) => typeof o !== 'undefined' && o
+  var defd = function (o) {
+    return typeof o !== 'undefined' && o;
```
Comment (davidmarkclements) [inline browser.js]: this will fail test (which includes linting)

File: browser.js [hunk 2]
```diff
@@ -307,9 +307,11 @@ function noop () {}
...
+  }
   try { 
-    if (typeof globalThis !== 'undefined') return globalThis
+    if (typeof globalThis !== 'undefined') return globalThis;
```
Comment (davidmarkclements) [inline browser.js]: stray ;

File: browser.js (hunk 1 above)
Comment (davidmarkclements) [reply browser.js]: (stray ;)

File: browser.js [hunk 3]
```diff
@@ -307,9 +307,11 @@ function noop () {}
 /* eslint-disable */
 /* istanbul ignore next */
 function pfGlobalThisOrFallback () {
-  const defd = (o) => typeof o !== 'undefined' && o
+  var defd = function (o) {
```
Comment (davidmarkclements) [inline browser.js]: may as well just do `function defd` and remove the var assign

File: browser.js (hunk 2 above)
Comment (dhalbrook) [reply browser.js]: Whitespace languages are a pet peeve, but let me change it :)

File: browser.js (hunk 3 above)
Comment (dhalbrook) [reply browser.js]: Done.

File: browser.js (hunk 2 above)
Comment (dhalbrook) [reply browser.js]: Done.

File: browser.js (hunk 1 above)
Comment (dhalbrook) [reply browser.js]: Fixed.

**Comment by dhalbrook** [comment]
Let me know if there are any other changes you need.

**Comment by davidmarkclements** [comment]
awesome!

**Comment by davidmarkclements** [comment]
released as v5.12.6

**Pull request merged**